index.js

```diff
@@ -0,0 +1,65 @@
+
+let playerObj = "http://localhost:3000/players/1"
```

Comment:
`playerObj` doesn't really make sense for a variable name for this string. Maybe `playerUrl`?